tests/incremental/04-manual-restart/03-partial_contexts.patch

```diff
@@ -0,0 +1,26 @@
+diff --git tests/incremental/04-manual-restart/03-partial_contexts.c tests/incremental/04-manual-restart/03-partial_contexts.c
+index a2a701673..d0b4d3efc 100644
+--- tests/incremental/04-manual-restart/03-partial_contexts.c
++++ tests/incremental/04-manual-restart/03-partial_contexts.c
+@@ -4,7 +4,7 @@ int foo(int x){
+ }
+ 
+ int main(){
+-    int x = 12;
++    int x = 13;
+     int y = foo(x);
+     assert(x == y);
+     return 0;
+diff --git tests/incremental/04-manual-restart/03-partial_contexts.json tests/incremental/04-manual-restart/03-partial_contexts.json
+index 99334fccb..3da6121a1 100644
+--- tests/incremental/04-manual-restart/03-partial_contexts.json
++++ tests/incremental/04-manual-restart/03-partial_contexts.json
+@@ -13,7 +13,7 @@
+             }
+         },
+         "restart_globs": {
+-            "globs": []
++            "globs": ["g"]
```

Comment:
There's no global `g` in this program, so I guess this is testing nothing?
Also might be useful to then have an error when no globals have the names given here.

Reply:
It should restart the start node of `foo`, so I changed the patch accordingly. When it is `g`, the assert is `Unknown`. 

For names that do not have a matching global in the CIL-file, it now also emits a warning.